### PR TITLE
Adding config properties for dashboard connections on seaclouds.yaml

### PR DESCRIPTION
**Pull request by mbarrientos**
Updating seaclouds.yaml according to changes introduced in https://github.com/SeaCloudsEU/SeaCloudsPlatform/pull/33

### CODE REVIEW

File: seaclouds.yaml [hunk 1]
```diff
@@ -28,8 +28,14 @@ services:
...
+  - serviceType: brooklyn.entity.webapp.jetty.Jetty6Server
     name: SeaClouds Dashboard 
     id: dashboard
     war: https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=dashboard&v=LATEST&e=war
     launch.latch: $brooklyn:component("deployer").attributeWhenReady("service.isUp")
+    brooklyn.config:
+      java.sysprops:
+        deployer.host: $brooklyn:component("deployer").attributeWhenReady("host.address")
+        deployer.httpPort: $brooklyn:component("deployer").attributeWhenReady("brooklynnode.webconsole.httpPort")
+        deployer.username: admin
```
Comment (andreaturli) [inline seaclouds.yaml]: I think it should be possible to read it from the `deployer` component. wdyt?

Comment (andreaturli) [inline seaclouds.yaml]: `$brooklyn:component("deployer").config("brooklynnode.managementUser")` and`$brooklyn:component("deployer").config("brooklynnode.managementPassword")` should help here ;)

Comment (mbarrientos) [inline seaclouds.yaml]: Cool. I was not sure about the reference, with the prefix "brooklynnode". I'm testing and fixing it

**Comment by andreaturli** [comment]
awesome @mbarrientos !
merging now

**Pull request merged**